instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/sessionreplay/imagebuttons/SrImageButtonsMaskUserInputTest.kt

```diff
@@ -26,11 +26,12 @@ internal class SrImageButtonsMaskUserInputTest :
         intentExtras = mapOf(SR_PRIVACY_LEVEL to SessionReplayPrivacy.MASK_USER_INPUT)
     )
 
-    @Test
+    @Ignore("Flakiness in CI, unsolved yet")
```

Comment:
Do we have a task created to address this?

Reply:
We have tasks to migrate the tests to the new pyramid yes